### PR TITLE
Adding flag for no default fields on report

### PR DESCRIPTION
**Pull request by jtroberts83**
Per a user request adding a flag --no_default_fields to the c7n-org reports function for cleaner output.

### CODE REVIEW

**Comment by jtroberts83** [comment]
I have tested and VOK that this code change works, pretty simple addition anyways :)

**Comment by jtroberts83** [comment]
@kapilt  Ready for your review and blessing.  Thanks!

File: tools/c7n_org/c7n_org/cli.py [hunk 1]
```diff
@@ -171,14 +171,15 @@ def report_account(account, region, policies_config, output_path, debug):
 @click.option('-s', '--output-dir', required=True, type=click.Path())
 @click.option('-a', '--accounts', multiple=True, default=None)
 @click.option('--field', multiple=True)
+@click.option('--no_default_fields', default=False, is_flag=True)
```
Comment (kapilt) [inline tools/c7n_org/c7n_org/cli.py]: generally try to avoid `_` in cli flags vs `-`

**Comment by kapilt** [comment]
thanks for the pr, one minor re exposed cli flag style, else looks good.

File: tools/c7n_org/c7n_org/cli.py (hunk 1 above)
Comment (jtroberts83) [reply tools/c7n_org/c7n_org/cli.py]: oh ok I will get that changed, thought the - was messing up the variable recognition but must have just been the way it was displayed.

File: tools/c7n_org/c7n_org/cli.py [hunk 2]
```diff
@@ -171,14 +171,15 @@ def report_account(account, region, policies_config, output_path, debug):
...
+@click.option('--no-default-fields', default=False, is_flag=True)
 @click.option('-t', '--tags', multiple=True, default=None)
 @click.option('-r', '--region', default=['us-east-1', 'us-west-2'], multiple=True)
 @click.option('--debug', default=False, is_flag=True)
 @click.option('-v', '--verbose', default=False, help="Verbose", is_flag=True)
 @click.option('-p', '--policy', multiple=True)
 @click.option('--format', default='csv', type=click.Choice(['csv', 'json']))
 @click.option('--resource', default=None)
-def report(config, output, use, output_dir, accounts, field, tags, region, debug, verbose, policy, format, resource):
+def report(config, output, use, output_dir, accounts, field, no-default-fields, tags, region, debug, verbose, policy, format, resource):
```
Comment (kapilt) [inline tools/c7n_org/c7n_org/cli.py]: not quite.. click automatically transforms `-` into `_` to be a valid python variable, so the exposed cli flag is --no-default-fields but the variable passed to the function is no_default_fields ..

Comment (jtroberts83) [reply tools/c7n_org/c7n_org/cli.py]: oh ok I get it, that makes sense now.  Thanks

**Pull request merged**